### PR TITLE
Add cluster etcd config to cluster.md.

### PR DESCRIPTION
**Pull request by wayilau**
Please answer these questions before submitting pull request

- Why submit this pull request?
- [ ] Bug fix
- [ ] New feature provided
- [ ] Improve performance
- [x] document

### CODE REVIEW

File: docs/en/setup/backend/backend-cluster.md [hunk 1]
```diff
@@ -90,4 +90,17 @@ cluster:
...
     # Nacos cluster nodes, example: 10.0.0.1:8848,10.0.0.2:8848,10.0.0.3:8848
     hostPort: ${SW_CLUSTER_NACOS_HOST_PORT:localhost:8848}
+```
+
+## Etcd
+Set the **cluster** module's implementor to **etcd** in
+the yml to active.
+
+```yaml
+cluster:
```
Comment (kezhenxu94) [inline docs/en/setup/backend/backend-cluster.md]: If you're adding docs of Etcd implementation of cluster plugin, then I think the snippet here is wrong, it's for dynamic configuration, I think those configuration for etcd cluster is:

```yml
  etcd:
    serviceName: ${SW_SERVICE_NAME:"SkyWalking_OAP_Cluster"}
#     etcd cluster nodes, example: 10.0.0.1:2379,10.0.0.2:2379,10.0.0.3:2379
    hostPort: ${SW_CLUSTER_ETCD_HOST_PORT:localhost:2379}
```

Comment (wayilau) [reply docs/en/setup/backend/backend-cluster.md]: sorry, i copied the wrong file.

Comment (wayilau) [reply docs/en/setup/backend/backend-cluster.md]: Hi, i've changed this.

**Pull request merged**